### PR TITLE
feat(trace-view): Update our mocks to create traces

### PR DESCRIPTION
**Pull request by wmak**
- This adds the ability to set spans & parent_span_id when creating sample transactions, which will help with tests on traces as well
- Currently mocked transactions are only connected by their trace id instead of properly through spans. Which means that new trace view functionality would appear not to work

### CODE REVIEW

File: bin/load-mocks [hunk 1]
```diff
@@ -748,6 +704,9 @@ def main(num_events=1, extra_events=False, load_transactions=False):
 
         OrganizationAccessRequest.objects.create_or_update(member=dummy_member, team=team)
 
+    if load_transactions:
```
Comment (markstory) [inline bin/load-mocks:707]: Any reason to not make this default to on? Having transactions by default simplifies onboarding and daily usage.

File: bin/load-mocks [hunk 2]
```diff
@@ -759,6 +718,131 @@ def main(num_events=1, extra_events=False, load_transactions=False):
...
+                            ),
+                            "offset": 0.02,
+                        },
+                        "span_id": backend_span_id,
+                        "trace_id": trace_id,
+                    }
+                    for name, backend_span_id in backend_span_ids
+                ],
+            )
+            if slow:
```
Comment (markstory) [inline bin/load-mocks:826]: When would you run this without slow?

File: bin/load-mocks (hunk 1 above)
Comment (Zylphrex) [reply bin/load-mocks:707]: From personal experience, loading transactions is slow and prone to crashing clickhouse even with the `slow` flag. So I would hold off on making this the default until that's addressed.

File: bin/load-mocks (hunk 2 above)
Comment (wmak) [reply bin/load-mocks:826]: I... will add some docs explaining this, but basically depends on your personal computer. I found that I could run `--load-transactions` consistently without crashing, but @Zylphrex has reported that running it crashes clickhouse consistently 😞

File: bin/load-mocks (hunk 1 above)
Comment (k-fish) [reply bin/load-mocks:707]: Wait really? 336 transactions will crash clickhouse locally? Maybe we can load a small handful by default to make at least something appear for onboarding like Mark mentioned, and only load the rest of them to create a trend as their own option.

**Review by k-fish — APPROVED** [review]
Looks good.

Not blocking, but as a mix of what Mark and Tony were talking about, I think we should probably include a small handful of transactions as a default, and offer the trend transactions or more comprehensive transaction mocks as their own option.

**Pull request merged**